### DISCUSSION TITLE
How do you use tags?

### DISCUSSION THREAD

**Discussion by amycole501** [uswds/uswds#5956] [Accessibility]
Tagging everyone in the community to give us your questions, comments, and examples of… well, [tags](https://designsystem.digital.gov/components/tag/)!

Whether you’re organizing content, tracking tasks, or something else entirely, we’d love to hear about how you use tags in your websites and applications. One of the big things we'd like to understand is whether you use interactive tags.

What types of tags do you use most frequently: interactive tags or static tags? In what cases do you use these tags?
If you use interactive tags, how do you determine whether they're coded as buttons or links? Or whether they're something else entirely?
Are there accessibility or usability implications to interactive tags?

**Comment by JonellaCulmer** [comment]
We use both interactive and static tags on our datatables. Here are examples of both interactive and static tags on our website: 

**Interactive:** Receipts datatable, tags used for filtering
https://www.fec.gov/data/receipts/?data_type=processed&committee_id=C00703975&two_year_transaction_period=2024&min_date=01%2F01%2F2023&max_date=12%2F31%2F2024

**Static:** FEC calendar, tags serve more as stylized categories and come with or without tooltips for additional explanation as needed.
https://www.fec.gov/calendar/

**Reply by amycole501** [reply]
I like how the URL changes depending on the tags selected. It makes it easy to send someone a specific data set!
I also like the combination of a checkbox and a tag; it seems intuitive as a dynamic filtering feature.

Have you had many people share feedback on the tags and how they use them?

**Comment by ryparker** [comment]
I'm bringing #6078 into this discussion so we can compare the agency examples. That issue describes a few different things:

- A removable token that clears a filter.
- A selectable filter or toggle.
- A tag-shaped link to another page or location.
- A static label that people can distinguish from an interactive control, visually and with assistive technology.

Let's choose one agency task first, describe its keyboard and screen-reader behavior, and see how far native buttons or links get us. Actions should use buttons rather than adding `role=button` to a noninteractive element; navigation and selection need the appropriate semantics too.

The examples and references in #6078 are still useful. We haven't decided to change the static tag, introduce a new component, or combine these variants in one API. Once we agree on an interaction and how to test it, we can open a focused implementation issue.
